openpype/settings/defaults/project_settings/webpublisher.json

```diff
@@ -0,0 +1,120 @@
+{
+    "publish": {
+        "CollectPublishedFiles": {
+            "task_type_to_family": {
+                "Animation": {
+                    "workfile": {
+                        "is_sequence": false,
+                        "extensions": [
+                            "tvp"
+                        ],
+                        "families": [],
+                        "tags": [],
+                        "subset_template_name": ""
```

Comment:
Is there a reason why to set subset template name here and not in `project_settings/global/tools/creator/subset_name_profiles`?

Reply:
Jezza had some reason, I was asking the same question.